framework/src/dgkernels/DGDiffusion.C

```diff
@@ -44,9 +44,7 @@ DGDiffusion::computeQpResidual(Moose::DGResidualType type)
 {
   Real r = 0;
 
-  const unsigned int elem_b_order = _var.order();
-  const double h_elem =
-      _current_elem_volume / _current_side_volume * 1. / Utility::pow<2>(elem_b_order);
+  const double h_elem = _current_elem->hmin();
 
```

Comment:
>_current_elem_volume to _current_elem->volume()
> _current_side_volume to _current_side->volume()

No, `Elem::volume()` is slow. We intentionally created the volume variables in `Assembly`.

Reply:
@YaqiWang  The question is which volume you want to use here. As @lynnmunday mentioned, the `libemsh->volme()` is the original element without considering any coordinate transformation. The `_current_element_volume` calculates the transformed volume. Which one makes more sense to use here?

Reply:
Yeah, I did not comment on this part. I tend to use the one with coordinate transform, but I am not sure. If we have to use the raw volume, we will have to create new variables in `Assembly`.

Reply:
A side note: we were thinking to have `InternalSideMaterial` (https://github.com/idaholab/moose/issues/14215) with which we can evaluate penalty coefficient. This could be useful when postprocessors require the same penalty coefficient. But MOOSE has not had this material yet.

Reply:
> Which one makes more sense to use here?

I would say to run a convergence study on it for an RZ problem. I suspect that it doesn't matter with respect to the polynomial error convergence order. But maybe the error constant would be different

Reply:
I would actually like to see a reference for the implementation we currently have in MOOSE. I thought that the variable-order based equation we use here was based off of Riviere's chapter one description of discretizing an elliptic equation, but I'm not seeing it. Without a reference, I'm more supportive of the change that @AhmedAlmetwally wants to make. However, it's not obvious to me why we would use `hmin()` over `hmax()`.

Reply:
@zachmprince could you review and approve changes.

Reply:
I'll dismiss this review as the PR does not change the behavior for a order >= 1 anymore.